src/components/MainComponent.js

```diff
@@ -17,6 +17,26 @@ import Debug from 'components/Debug'
 import { isChromeExtensionBackground } from 'services/notificationService'
 import Notifications from 'components/Extension/Notifications'
 
+try {
+  const Checklist = require('components/Checklist')
+}
+catch(error) {
+  if ( !error.toString().includes('Cannot find module') ) {
```

Comment:
I believe you can access the error code here:
`if (error.code && error.code === 'MODULE_NOT_FOUND') {`